### DISCUSSION TITLE
Confusion on LazyVim opt and opts

### DISCUSSION THREAD

**Discussion by NoahELE** [LazyVim/LazyVim#465] [General]
```lua
require("lazy").setup({
  spec = {
    -- add LazyVim and import its plugins
    {
      "LazyVim/LazyVim",
      import = "lazyvim.plugins",
      opt = { -- HERE!!!
        colorscheme = "default",
      },
    },
```

I wonder why the marked place is `opt` instead of `opts`, and where can I find the spec for it.

**Comment by folke** [comment]
It should be `opts`, where did you read that it should be `opt`?

LazyVim options are documented here https://www.lazyvim.org/configuration#default-settings

**Reply by NoahELE** [reply]
Thanks, I figure out that I somehow modify the line and thought it was the original code.
